scripts/variantstore/scripts/run_in_hail_cluster.py

```diff
@@ -64,7 +65,9 @@ def run_in_cluster(cluster_name, account, worker_machine_type, master_machine_ty
         )
 
         # prepare custom arguments
-        secondary_script_path_arg = f'--py-files {" ".join(secondary_script_path_list)}' if secondary_script_path_list else ''
+        # the following says `--py-files` is supposed to be a comma separated list
```

Comment:
How did this work before? Did we never pass multiple py-files?

Reply:
lol it did work before and it hasn't been cleaned up completely -- right? dont you want to remove line 81 as well?

Reply:
yes exactly, I was the lucky first person to supply more than one